book/src/ed_attack_vectors.md

```diff
@@ -0,0 +1,11 @@
+## 5. Attack Vectors
```

Comment:
You don't need that `5.` in there. `mdbook` adds those automatically. Doing it by hand will only make it harder to create crosslinks.

Reply:
Ah I see. will clean up in next update